filters/auth/forwardtoken.go

```diff
@@ -62,6 +76,17 @@ func (f *forwardTokenFilter) Request(ctx filters.FilterContext) {
 		return
 	}
 
+	if len(f.StripJsonKeys) > 0 {
+		switch typedTiMap := tiMap.(type) {
+		case map[string]interface{}:
+			removeKeys(typedTiMap, f.StripJsonKeys)
```

Comment:
`fallthrough` seems to be the better option here

Reply:
compiler is complaining that `fallthrough` isn't supported for type switch.

Reply:
ah ok, my parser was not good enough :)